docs/manual/src/proc_macro/index.md

```diff
@@ -138,6 +138,31 @@ fn do_something(foo: MyFooRef) {
 }
 ```
 
+### Renaming functions, methods and constructors
+
+A single exported function can specify an alternate name to be used by the bindings by specifying a `name` attribute.
+
+```rust
+#[uniffi::export(name = "something")]
```

Comment:
This is nice! Any plans to support renaming of `args`? That would be really fantastic! In the realm of Swift, we typically use prepositions a lot. so `from, `to`, `by` etc is the preferred label, e.g. [see Swift standard library `stride:from:to:by`](https://developer.apple.com/documentation/swift/stride(from:to:by:)). But since Rust does not have the notion of seperated "labels" (externally shown) and argname (internal to the body of the function) like Swift does, as Rust devs it is strange to use `from`, `to`, `by` as arg names.

So it would be amazing of `uniffi::export(name` could support renames of args!

Bikeshedding syntax:
```rust
#[uniffi::export(fn_name = "stride", arg_names = "from,to,by")]
fn do_stride(start: usize, end: usize, stride_length: usize) {
}
```

Would that be interesting to add support for?

Would it be possible?